decidim-proposals/app/controllers/decidim/proposals/proposal_endorsements_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # Exposes the proposal endorsement resource so that users can endorse proposals.
+    class ProposalEndorsementsController < Decidim::Proposals::ApplicationController
+      include ProposalEndorsementsHelper
+      include Rectify::ControllerHelpers
```

Comment:
Instead of using this, please use our own `present` helper:

https://github.com/decidim/decidim/blob/eae6367db40e3d7d6656a4144f7f540ac25bd0a6/decidim-core/app/helpers/decidim/application_helper.rb#L28-L34

This was added by @deivid-rodriguez at #2360, although we might lack some docs there.

Reply:
@mrcasals Sorry, I'm not sure what you mean. You say to use a presenter instead of what?
I've removed the 2 includes as they are not required. Maybe it was this that was confusing you.

Reply:
We have our own `present` method, defined in the file I linked. If you added the `Rectify::ControllerHelpers` module to use that method, then don't, as our own `present` behaves differently from the one y `rectify`.